index.rst

```diff
@@ -1,57 +1,343 @@
+For instance, when operators need to prepare the observatory for calibrations or for night operation, they will add a rather complex script to the ScriptQueue that will perform all the required tasks (e.g. position telescope and dome, perform required fine tuning in positioning, open mirror covers and so on).
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+The Scheduler for the survey essentially adds scripts to the scriptQueue after each visit until told by the observer to stop.
 
-  Use the following syntax for sections:
+As mentioned already, interacting with the ScriptQueue is mainly done through :ref:`LOVE ScriptQueue user interface <fig-scriptqueueui>`.
+The interface allows users to stop, start and pause the ScriptQueue.
+It also displays the status of scripts and any errors that occur.
+Shuffling the order of queued scripts is also possible.
+Lastly, users can quickly relaunch previously run scripts without having to re-enter any modifications to the default configuration.
 
-  Sections
-  ========
+.. figure:: /_static/ScriptQueueUI.png
+   :name: fig-scriptqueueui
+   :target: ../_images/ScriptQueueUI.png
+   :alt: LOVE ScriptQueue user interface
 
-  and
+   A screenshot of the LOVE interface to interact with the ScriptQueue.
 
-  Subsections
-  -----------
+In general, scripts require minimum interaction to be executed except, of course, for the occasional configuration.
+A database of configurations will be available for users to execute the most common set of operations.
+Users also have the possibility to edit configurations from the user interface, which also provides on-the-fly schema validation.
 
-  and
+In general, using the ScriptQueue requires some familiarity with the observatory system and minimal set of software development skills.
+Users should be able to, after inspecting the LOVE status screens, determine the state of the observatory and its readiness to perform certain types of operations.
+Other than that, some knowledge about `yaml`_ and `json schema`_ may be useful for writing and inspecting script configurations, though :ref:`LOVE <fig-scriptqueueui>` will eventually provide features to help understanding and validating them prior to execution.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+.. _yaml: https://yaml.org/spec/1.2/spec.html
+.. _json schema: http://json-schema.org
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+.. _SAL_Scripts:
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+SAL Scripts
+-----------
 
-     Caption text.
+`SAL Scripts`_ are the files which contain the logic and coordination of events and CSCs that get executed by the `ScriptQueue`_.
+It is not expected these will be modified during standard night-time operations.
+All `SAL Scripts`_ vary greatly in complexity of the operation being performed.
```

Comment:
Maybe start a new paragraph?

suggested edit:

> SAL Scripts vary considerably in complexity depending on the operation being performed.

Reply:
Don't need a new paragraph... it's only the 3rd sentence... but I edited the phrase.